doc/src/manual/calling-c-and-fortran-code.md

```diff
@@ -97,7 +97,7 @@ julia> (Cstring,)
 
 In practice, especially when providing reusable functionality, one generally wraps [`ccall`](@ref)
 uses in Julia functions that set up arguments and then check for errors in whatever manner the
-C or Fortran function indicates them, propagating to the Julia caller as exceptions. This is especially
+C or Fortran function indicates them. And if an error occurs it is thrown as a normal Julia exception. This is especially
```

Comment:
"indicates them" -> "specifies"

"And if an error occurs it" -> "If an error occurs, it"

Reply:
That would subtly change the sense. There's nothing wrong with starting a sentence with 'and' esp. if if clarifies.